src/styles/components/_menuheader.scss

```diff
@@ -0,0 +1,24 @@
+.lqe-hdr-menu {
```

Comment:
Could this be moved into the following lqe-hrd-menu class?
Also will it work if child classes be prefixed with & instead of the full name so for example &__title-list instead of lqe-hdr-menu_title-list?

Reply:
Xin told me to do it like this.  First one is setting variables and second one is actual styling.